project_event/models/calendar_event.py

```diff
@@ -21,3 +21,17 @@ class CalendarEvent(models.Model):
         string='Client Type',
     )
     color = fields.Char(related='category_id.color')
+    client_tag = fields.Many2one(
```

Comment:
If it is a readonly field it is better to put it as a related field

Reply:
Right, done